nodes/scenenet_to_rosbag.py

```diff
@@ -407,6 +413,28 @@ def publish(scenenet_path, trajectory, output_bag, to_frame):
     args = parser.parse_args()
     if args.scenenet_path:
         scenenet_path = args.scenenet_path
+    if args.protobuf_path:
+        protobuf_path = args.protobuf_path
+    else:
+        # Default
+        protobuf_path = os.path.join(scenenet_path, 'data/scenenet_rgbd_val.pb')
+        print("Path to protobuf file not found as an argument (use "
+              "-protobuf_path to indicate one). Using default value "
+              "{}".format(protobuf_path))
+    if args.dataset_type:
+        # Check that dataset_type is valid
+        if args.dataset_type in ("train", "val"):
```

Comment:
Use the [native](https://docs.python.org/2.7/library/argparse.html#choices) _choices_ feature of ArgumentParser, it will take care of the error message as well.

Reply:
Fixed in 3aa93ce26c031f252925fc88f09be3bb8ccdd667.